src/test/scala/nl/knaw/dans/lib/logging/servlet/ServletLoggerSpec.scala

```diff
@@ -182,6 +186,24 @@ class ServletLoggerSpec extends FlatSpec with Matchers with MockFactory with Emb
     }
   }
 
+  it should "call the renderUncaughtException when an exception is thrown in the servlet route" in {
+    val serverPort = localPort.fold("None")(_.toString)
+
+    (() => mockedLogger.isInfoEnabled()) expects() once() returning true
+    (() => mockedLogger.isWarnEnabled()) expects() once() returning true
```

Comment:
I had a hard time to tell the forest from the trees. Please extract some generic/better readble methods that does the mocking.

What goes below is essential for the test. How many times `isXxxEnabled` is called does not seem essential to me: use `anyTimes`. Anyway extract a method `expectingIsEnabled`, if you think the counts are essential use an int argument per `Xxx`.

Also extract methods `expectingLogXxxMessage` (or `expectingLogMessage` with Xxx as argument) to get at least the following out of the equation:
```
    (mockedLogger.info(_: String)) expects where {
    } once()
```

Reply:
Thanks for the feedback! I think 4b1b5ee really improved the tests' readability!